apps/prairielearn/elements/pl-sketch/info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "controller": "pl-sketch.py",
+  "dependencies": {
+    "nodeModulesScripts": ["sketchpad/scripts/sketchpad.js"],
```

Comment:
This element appears to be almost entirely unmaintained; it hasn't seen a release in 7 years and appears to be somewhat limited in what it can do. Did you look into any other options?

That said, the source code isn't too long (~300 lines) and is MIT licensed, so we should be covered if we decide we need to fork it to support more use cases.

Reply:
I'm not married to this package at all and have not done any research into other options... will try and look around but I don't have much capacity to make comprehensive changes at the moment, but on the other hand I'd also love additional functionality (annotating on pictures, button to wipe the canvas, download the sketch as a png, instructor annotations during manual grading, etc... the list can be long!).